package.json

```diff
@@ -13,7 +13,8 @@
     "test": "jest",
     "test:watch": "npm test -- --watch",
     "build": "npm run build:js && npm run build:sass",
-    "build:fonts": "mkdir -p dist/fonts && cp -R `node_modules`/font-awesome/fonts/* `node_modules`/npm-font-open-sans/fonts/* dist/fonts/",
+    "build:prod": "browserify --extension=jsx src/js/App.jsx -t [ babelify ] -g [ envify --NODE_ENV production ] -g uglifyify | uglifyjs --compress --mangle > dist/cloud-explorer.js",
```

Comment:
Why mangle?

Reply:
I used React recommendations for Browserify https://reactjs.org/docs/optimizing-performance.html#browserify

Reply:
Ok then, if it works then :)